go.mod

```diff
@@ -26,9 +27,22 @@ require (
 )
 
 replace (
+	github.com/metal3-io/baremetal-operator => github.com/openshift/baremetal-operator v0.0.0-20200715132148-0f91f62a41fe
+	github.com/metal3-io/cluster-api-provider-baremetal => github.com/openshift/cluster-api-provider-baremetal v0.0.0-20190821174549-a2a477909c1d
```

Comment:
Why we need them?

Reply:
Because the mch operator depends on theses, removing these will report errors.